ADAL/src/telemetry/ADTelemetryAPIEvent.h

```diff
@@ -30,6 +30,7 @@
 - (void)setExtendedExpiresOnSetting:(NSString*)extendedExpiresOnSetting;
 - (void)setPromptBehavior:(ADPromptBehavior)promptBehavior;
 - (void)setUserInformation:(ADUserInformation*)userInfo;
+- (void)setUserId:(NSString*)userId;
```

Comment:
Yeah. The comments I made are to match the style used in Apple's headers. I try to clean up files when I'm around them, but yes, there are still places that need to be cleaned.